server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -637,5 +637,16 @@ export function configureWebSocketServices(
 			}
 			disposers.splice(0, disposers.length);
 		});
+
+		socket.on(
```

Comment:
The PR description says the server part is out of scope for this PR. Is the description outdated, or this change slipped in?

Reply:
Sorry for the confusion, updated the description